### ISSUE TITLE
Wrong path for v:uri.image with absolute path and absRefPrefix

### ISSUE THREAD

**Issue by SventB** [FluidTYPO3/vhs#936]
(TYPO3 version 7.5, vhs version 2.3.3)

With e.g. 
```
config.absRefPrefix = Web/
```
and the Fluid viewhelper
```
<v:uri.image src="fileadmin/my/image.jpg" relative="false" />
```
the result is http://www.mydomain.com/Web/Web/fileadmin/my/image.jpg (there is "Web" two times).

That's why in function preprocessImage() the absRefPrefix is added:
```php
$this->mediaSource = $GLOBALS['TSFE']->absRefPrefix . GeneralUtility::rawUrlEncodeFP($publicUrl);
```
and in some steps later, in function preprocessSourceUri(), again:
```php
$src = GeneralUtility::getIndpEnv('TYPO3_SITE_URL') . ltrim($src, '/');
```
(TYPO3_SITE_URL contains the absRefPrefix)

**Comment by hhoechtl** [comment]
Already fixed in 2fb89e3

Can be closed

**Comment by bjo3rnf** [comment]
Thanks @hhoechtl for the hint.
